src/frontend/components/SøknadsSteg/1-OmDeg/OmDeg.tsx

```diff
@@ -47,7 +47,7 @@ const JaNeiBolk: React.FC<IJaNeiBolkProps> = ({ skjema, felt, spørsmålTekstId
 
 // TODO: Bruk konstant for mobilbredde
```

Comment:
Kan vi ta bort denne kommentaren nå da? :D

Reply:
Generelt egentlig, kanskje søke på mobil/mobile for å se om det henger igjen flere kommentarer?